doc/guides/9 - Presenters/1 - Menu Presenter.textile

```diff
@@ -41,31 +42,48 @@ Run the migration:
 rake db:migrate
 </shell>
 
-h3. Decorating the Refinery::Page model
+h3. Decorating the +Refinery::Page+ model
 
-Before overriding Refinery's form view, we want to decorate the +Refinery::Page+ class. Create a file +app/decorators/models/refinery/page_decorator.rb+ with this content:
+We want to decorate the +Refinery::Page+ class. Create a file +app/decorators/models/refinery/page_decorator.rb+ with this content:
 
 <ruby>
 Refinery::Page.class_eval do
-  attr_accessible :show_in_footer
-
   def self.footer_menu_pages
-    where :show_in_footer => true
+    where show_in_footer: true
   end
 end
 </ruby>
 
-We added +show_in_footer+ to the allowed attributes list so that it doesn't raise a mass-assignment error each time someone tries to save the page. We also added +footer_menu_pages+ class method to abstract away ActiveRecord query method.
+We added +footer_menu_pages+ class method to abstract away ActiveRecord query method.
+
+h3. Allow +show_in_footer+ attribute in +Refinery::Admin::PagesController+
+
+Before overriding Refinery's form view, we want to decorate the +Refinery::Admin::PagesController+ class. Create a file +app/decorators/controllers/refinery/admin/pages_controller_decorator.rb+ with this content:
+
+<ruby>
+Refinery::Admin::PagesController.class_eval do
+  # work around from https://github.com/refinery/refinerycms-page-images/blob/master/app/decorators/controllers/refinery/admin/pages_controller_decorator.rb
+  if self.instance_methods.exclude?(:page_params_with_show_in_footer_params)
+    def page_params_with_show_in_footer_params
+      show_in_footer_params = params.require(:page).permit(:show_in_footer)
+    end
+
+    alias_method_chain :page_params, :show_in_footer_params
+  end
+end
```

Comment:
This workaround shouldn't be required since Decorators 2.0.0 was released, and you're ignoring all of the params that were previously allowed.  Try this:

```ruby
def page_params_with_show_in_footer
  page_params_without_show_in_footer.permit(:show_in_footer)
and
alias_method :page_params, :show_in_footer
```